src/Commands/User.php

```diff
@@ -0,0 +1,693 @@
+        $validation = Services::validation();
+        $validation->setRules($this->validationRules);
+
+        if (! $validation->run($data)) {
+            foreach ($validation->getErrors() as $message) {
+                $this->write($message, 'red');
+            }
+
+            $this->write('User creation aborted', 'red');
+
+            throw new RuntimeException('User creation aborted');
+        }
+
+        $userModel = model(UserModel::class);
+
+        $user = new UserEntity($data);
+        $userModel->save($user);
+
+        $this->write('User "' . $username . '" created', 'green');
+    }
+
+    /**
+     * Activate an existing user by username or email
+     *
+     * @param string|null $username User name to search for (optional)
+     * @param string|null $email    User email to search for (optional)
+     */
+    private function activate(?string $username = null, ?string $email = null): void
+    {
+        $user = $this->findUser('Activate user', $username, $email);
+
+        $confirm = $this->prompt('Activate the user ' . $user->username . ' ?', ['y', 'n']);
+
+        if ($confirm === 'y') {
+            $userModel = model(UserModel::class);
+
+            $user->active = 1;
+            $userModel->save($user);
+
+            $this->write('User "' . $user->username . '" activated', 'green');
+        } else {
+            $this->write('User "' . $user->username . '" activation cancelled', 'yellow');
+        }
+    }
+
+    /**
+     * Deactivate an existing user by username or email
+     *
+     * @param string|null $username User name to search for (optional)
+     * @param string|null $email    User email to search for (optional)
+     */
+    private function deactivate(?string $username = null, ?string $email = null): void
+    {
+        $user = $this->findUser('Deactivate user', $username, $email);
+
+        $confirm = $this->prompt('Deactivate the user "' . $username . '" ?', ['y', 'n']);
+
+        if ($confirm === 'y') {
+            $userModel = model(UserModel::class);
+
+            $user->active = 0;
+            $userModel->save($user);
+
+            $this->write('User "' . $user->username . '" deactivated', 'green');
+        } else {
+            $this->write('User "' . $user->username . '" deactivation cancelled', 'yellow');
+        }
+    }
+
+    /**
+     * Change the name of an existing user by username or email
+     *
+     * @param string|null $username    User name to search for (optional)
+     * @param string|null $email       User email to search for (optional)
+     * @param string|null $newUsername User new name (optional)
+     */
+    private function changename(
+        ?string $username = null,
+        ?string $email = null,
+        ?string $newUsername = null
+    ): void {
+        $validation = Services::validation();
+        $validation->setRules([
+            'username' => 'required|is_unique[users.username]',
```

Comment:
Get table name from config file.

Reply:
These code are not needed. I will remove.